src/Interfaces/AbstractZonotope.jl

```diff
@@ -419,6 +419,18 @@ https://math.stackexchange.com/q/3356460
 To avoid cumulative sum from both directions separately, we build a 2d index matrix
 to sum generators for both directions in one matrix-vector product.
 """
+
+@inline function _angles(point::AbstractVector{N})  where{N<:Real}
```

Comment:
I think you need to keep this docstring attached to the `vertices_list` method.

Also, is the algorithm still the same or would you update the docstring?